controllers/release/release_adapter_test.go

```diff
@@ -491,13 +472,27 @@ var _ = Describe("Release Adapter", Ordered, func() {
 		link, err := adapter.getReleasePlan()
 		Expect(err).Should(Succeed())
 		Expect(reflect.TypeOf(link)).To(Equal(reflect.TypeOf(&appstudiov1alpha1.ReleasePlan{})))
+	})
 
-		// It should err when the ReleasePlan does not exist
+	It("ensures the target context returns an error when no releaseplan exists", func() {
```

Comment:
Can we use the change to  rewrite the description? Something like this:
```suggestion
	It("ensure an error is returned when setting the target context with a missing releaseplan", func() {
```

Reply:
sure.